source/experimental/ConsoleRenderer/ConsoleRenderer/ConsoleRender.cpp

```diff
@@ -12,15 +12,16 @@
 using namespace AdaptiveCards;
 using namespace std::string_literals;
 
-size_t RenderElements(std::vector<std::shared_ptr<BaseCardElement>> elements, size_t columnWidth, size_t x, size_t y, std::vector<std::string>& consoleString);
-size_t RenderColumnSet(ColumnSet &set, size_t columnWidth, size_t x, size_t y, std::vector<std::string> &consoleString);
-size_t RenderColumn(Column &set, size_t columnWidth, size_t x, size_t y, std::vector<std::string> &consoleString);
-size_t RenderFactSet(FactSet &set, size_t FactWidth, size_t x, size_t y, std::vector<std::string> &consoleString);
-size_t RenderFact(Fact &set, size_t FactWidth, size_t x, size_t y, std::vector<std::string> &consoleString);
+size_t RenderElements(
```

Comment:
We should just delete the `source/experimental` directory altogether :)

Reply:
I can do it in a separate PR